react/features/toolbox/functions.web.js

```diff
@@ -138,6 +138,19 @@ export function getToolbarClassNames(props: Object) {
 }
 
 /**
+ * Indicates if a toolbar button is enabled.
+ *
+ * @param {string} name - The name of the setting section as defined in
+ * interface_config.js.
+ * @returns {boolean} - True to indicate that the given toolbar button
+ * is enabled, false - otherwise.
+ */
+export function isButtonEnabled(name) {
```

Comment:
I refactored a little bit UIUtils in order to decrease the size of always on top bundle. Specifically `isButtonEnabled` was a good candidate to be moved in the toolbox feature. You think it is better to be defined in UIUtils or somewhere else?